gr-dtv/include/gnuradio/dtv/atsc_plinfo.h

```diff
@@ -0,0 +1,157 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2001,2006,2014 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef DTV_INCLUDED_ATSC_PLINFO_H
+#define DTV_INCLUDED_ATSC_PLINFO_H
+
+#include <gnuradio/dtv/atsc_consts.h>
+#include <cassert>
+#include <cstring>
+
+namespace gr {
+namespace dtv {
+
+/*!
+ * \brief pipeline info that flows with data
+ *
+ * Not all modules need all the info
+ */
+
+struct plinfo_data {
```

Comment:
so this is a new one, but it's not used by `plinfo`? I'm not sure I follow.

Reply:
The idea here was to break out the data portion of the class into a struct.  It is only this portion that gets sent along on the tags now.